### ISSUE TITLE
finetune SqueezeLLM

### ISSUE THREAD

**Issue by kiucho** [SqueezeAILab/SqueezeLLM#20]
Hi. First of all, thank you for sharing your great work.
I am looking for a way to `fine-tune LLaMA-7B model` with SqueezeLLM. 
It seems that the publicly available code in `llama.py` can only `benchmark` or `eval`.
Do I need a special way to fine-tune SqueezeLLM or can I train it with normal pytorch fine-tuning?

**Comment by kssteven418** [comment]
Our method is a post-training quantization method that does not require any finetuning. 
Supporting fine-tuning would require a straight-through estimator (STE) that allows the gradient to pass through the quantization operators, but we do not have an implementation for it.
